great_expectations/expectations/expectation.py

```diff
@@ -1506,7 +1506,6 @@ class BatchExpectation(Expectation, ABC):
     batch_id: Union[str, None] = None
     row_condition: Union[str, None] = None
     condition_parser: Union[str, None] = None
-    mostly: Mostly = 1.0
```

Comment:
This was wrong. `mostly` does not apply to non-map Expectations, but those Expectations inherit from `BatchExpectation`.